Ops handbook tools (#832)
publiccodenet/about: activities/tool-management/voys.md

## Using Voys

> Employees that need to call regularly receive an account. A local (020) number will be assigned. Voys can be used with an app on a mobile phone. [We have a guide on how to make and receive calls from your mobile phone using the Voys app.](../communication/using-phone-app.md).

Reviewer: ```suggestion
Employees that need to call regularly receive an account. A local (020) number will be assigned. Voys can be used with an app on a mobile phone. Read the [guide on how to make and receive calls from your mobile phone using the Voys app](../communication/using-phone-app.md).
```
